server/database/queries/users/addUsersQuery.js

```diff
@@ -0,0 +1,12 @@
+const connection = require('../../data/connection');
+
+const addUsersQuery = (userName, email, password) => {
+  const sql = {
+    text:
+      'INSERT INTO users(user_name,email,password) VALUES($1 ,$2 ,$3) returning *;',
```

Comment:
The `*` will return everything even the hashed password which is something we don't want
```suggestion
      'INSERT INTO users(user_name,email,password) VALUES($1 ,$2 ,$3) returning id, user_name, email;',
```